features/steps/test_starter/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -6,7 +6,7 @@ isort~=5.0
 jupyter~=1.0
 jupyterlab_server>=2.11.1, <2.16.0
 jupyterlab~=3.0, <3.6.0
-kedro[pandas.CSVDataSet]=={{ cookiecutter.kedro_version }}
+kedro-datasets[pandas.CSVDataSet]~=1.2
```

Comment:
This isn't actually related to the move to `pyproject.toml` right? It's not a big issue to add it here, but in general I think we shouldn't be mixing changes in PRs that aren't actually related.

Reply:
Fair enough, I'm going to drop this commit and open an issue about it

Reply:
https://github.com/kedro-org/kedro/issues/2933